apps/provider-proxy/src/container.ts

```diff
@@ -13,9 +14,15 @@ const providerService = new ProviderService((network: SupportedChainNetworks) =>
   // @see https://github.com/mswjs/msw/discussions/2416
   return process.env.TEST_CHAIN_NETWORK_URL || netConfig.getBaseAPIUrl(network);
 }, fetch);
-const providerProxy = new ProviderProxy(Date.now, providerService);
+const certificateValidator = new CertificateValidator(
```

Comment:
Great, we will be able to use it in the indexer as well!

Reply:
Just for context, in the indexer we have scheduled tasks where we query the providers periodically using the status and version endpoint. https://github.com/akash-network/console/blob/main/apps/indexer/src/index.ts#L98